tests/pipelines/test_train_step.py

```diff
@@ -134,7 +134,11 @@ def test_train_steps_autologs(tmp_pipeline_root_path):
 
 def test_train_steps_with_correct_tags(tmp_pipeline_root_path):
     with mock.patch.dict(
-        os.environ, {_MLFLOW_PIPELINES_EXECUTION_DIRECTORY_ENV_VAR: str(tmp_pipeline_root_path)}
+        os.environ,
+        {
+            _MLFLOW_PIPELINES_EXECUTION_DIRECTORY_ENV_VAR: str(tmp_pipeline_root_path),
+            "target_step_name": "train",
```

Comment:
What about defining this env var key as `_MLFLOW_PIPELINES_TARGET_STEP_NAME_ENV_VAR` (following the pattern above)?

Reply:
+1

Reply:
good point, done!